war.js

```diff
@@ -0,0 +1,50 @@
+var values  = [2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K", "A"];
+var suits   = ["Clubs", "Diamonds", "Hearts", "Spades"];
+
+function shuffle(array){
+  var currentIndex = array.length;
+  var randomIndex;
+  var temporaryValue;
+  while(currentIndex > 0){
+    randomIndex   = Math.floor(Math.random() * currentIndex);
+    currentIndex -= 1;
+
+    temporaryValue       = array[currentIndex];
+    array[currentIndex]  = array[randomIndex];
+    array[randomIndex]   = temporaryValue;
+  }
+  return array;
+}
+var i = 0
+function buildDeck(sAry, vAry){
```

Comment:
You don't really need to pass in `values` and `suits`. Remember functions can access all the variables outside them.